sig-scalability/slos/dns_programming_latency.md

```diff
@@ -1,10 +1,10 @@
-## Network programming latency SLIs/SLOs details
+## DNS programming latency SLIs/SLOs details
 
 ### Definition
 
 | Status | SLI | SLO |
 | --- | --- | --- |
-| __WIP__ | Latency of programming a single in-cluster dns instance, measured from when service spec or list of its `Ready` pods change to when it is reflected in that dns instance, measured as 99th percentile over last 5 minutes | In default Kubernetes installation, 99th percentile of (99th percentiles across all dns instances) per cluster-day <= X |
+| __WIP__ | Latency of programming dns instance, measured from when service spec or list of its `Ready` pods change to when it is reflected in that dns instance, measured as 99th percentile over last 5 minutes aggregated across all dns instances | In default Kubernetes installation, 99th percentile of per cluster-day <= X |
```

Comment:
The last part of this is unclear: "99th percentile of per cluster-day <= X"

Reply:
So to be clear: All samples go into one large pool, and SLI is p99 of all of them?  SGTM.

Reply:
Is X a fraction of total good minutes?

Reply:
> Is X a fraction of total good minutes?

Yes - that was my intention. Will clarify.

> All samples go into one large pool, and SLI is p99 of all of them? SGTM.

Yes exactly - I will add a footnote for it.

Reply:
Footnote added.

The clarification about "fraction of good minutes" was added once for all slos in slos.go:
https://github.com/kubernetes/community/pull/3307/files#diff-e0acb8848e18cc21aa7dbc56ee9e892dR114